docs/guides/02-add-data-sources.md

```diff
@@ -16,60 +16,46 @@ By the end of this guide, you will be able to create this CARTO VL map:
+> In fact, there is a fourth type of source in CARTO VL called [carto.source.MVT](/developers/carto-vl/reference/#cartosourcemvt) but its usage is only recommended for some advanced cases.
 
 ### Getting started
-In the following section you will see how to use three main source types. Before getting started with that, you will need to create a basic map.
+In the following section you will see how to use the three main source types. Before getting started with that, you will need to create a basic map.
 
 You can start from this [basemap](/developers/carto-vl/examples/maps/guides/getting-started/step-1.html). Go ahead and copy its source code into a new file called `sources.html`.
 > To copy the source code from an example, you just have to navigate to it with your browser, click right button > `View source` and copy the whole text, from `<html>` to `</html>` tags.
 
-Next, to warm up, add navigation controls, just after the map creation:
-```js
-const nav = new mapboxgl.NavigationControl();
-map.addControl(nav, 'top-left');
-```
-
 ### Dataset source
-A `Dataset` can be managed using [carto.source.Dataset](/developers/carto-vl/reference/#cartosourcedataset). It is a source with geographic information related to a specific topic (such as *stores*, *streets* or *counties*). If you have a GIS background, this is like a local vector file with points, lines or polygons, hosted in CARTO. If you don't, you can think of it as a table on the server with a geometry field you can use to draw it on a map.
+A `Dataset` source can be created with the [carto.source.Dataset](/developers/carto-vl/reference/#cartosourcedataset) constructor. If you have a GIS background, this is like a local vector file with points, lines or polygons, hosted in CARTO. If you don't, you can think of it as a table on the server with a geometry field you can use to draw it on a map.
 
 #### Add a Dataset source
 You already know how to add a `Dataset` thanks to the *Getting Started* guide:
 ```js
 const aSource = new carto.source.Dataset('name_of_your_dataset');
 ```
-That was using `carto.setDefaultAuth` method, but now you will see how to include custom credentials for a specific dataset. Add this to your current working file (*sources.html* if you followed our suggestion), just after map creation.
+That was using `carto.setDefaultAuth` method, but now you will see how to include custom credentials for a specific dataset. This allows to work with multiple Sources with different combinations of usernames and API keys. Add this to your current working file (*sources.html* if you followed our suggestion), just after map creation.
 ```js
 const citiesSource = new carto.source.Dataset('populated_places', {
     username: 'cartovl',
     apiKey: 'default_public'
 });
 ```
 
-As with any `Source`, you should then pass it to a `Layer` to visualize it, but first let's create its Viz with a simple style:
```

Comment:
Although not related to the 'sources' I won't remove this, as it is needed to keep the styles as desired

Reply:
We could leave the code in the example but not talk about it, right?

Reply:
Fair enough for me.